pkg/gatherers/clusterconfig/machine_sets.go

```diff
@@ -18,7 +18,7 @@ import (
 // GatherMachineSet collects MachineSet information
 //
 // The Kubernetes api:
-//       https://github.com/openshift/machine-api-operator/blob/master/pkg/generated/clientset/versioned/typed/machine/v1beta1/machineset.go
+//       https://github.com/openshift/api/blob/master/machine/v1beta1/types_machineset.go#L21
```

Comment:
I think we don't use these line numbers (there's good chance that it will change in the future right) so please link only the file. Thanks